foundation/foundation.css

```diff
@@ -0,0 +1,133 @@
+body {
+    font-family: Arial,Helvetica,Verdana,sans-serif;
+    font-size: 15px;
+    line-height: 22px;
+    color: #666666;
+}
+
+h1 {
+    font-size: 32px;
+    line-height: 36px;
+    color: #333333;
+    padding-bottom: 18px;
+}
+
+h2 {
+    font-size: 24px;
+    line-height: 22px;
+    color: #333333;
+    padding-bottom: 14px;
+}
+
+h3 {
+    font-size: 18px;
+    line-height: 22px;
+    color: #333333;
+    padding-bottom: 6px;
+}
+
+p {
+	  color: ;
```

Comment:
empty color and missaligned

Reply:
the color is in the code, looks like a bug from the code editor